test/spec/modules/reloadBidAdapter_spec.js

```diff
@@ -0,0 +1,300 @@
+import { expect } from 'chai';
+import { spec } from 'modules/reloadBidAdapter';
+import { newBidder } from 'src/adapters/bidderFactory';
```

Comment:
Remove unused imports. Following are not used in your test file anywhere
```
import { newBidder } from 'src/adapters/bidderFactory';
import * as bidderFactory from 'src/adapters/bidderFactory';
import { deepClone } from 'src/utils';
import { config } from 'src/config';
```

Reply:
Fixed